python-avd/pyavd/_eos_designs/structured_config/network_services/ethernet_interfaces.py

```diff
@@ -308,7 +319,12 @@ def _set_subif_parent_interfaces(self: AvdStructuredConfigNetworkServicesProtoco
             interface = EosCliConfigGen.EthernetInterfacesItem(
                 name=interface_name,
                 shutdown=False,
+                metadata=EosCliConfigGen.EthernetInterfacesItem.Metadata(
+                    peer_type="l3_interface",
+                    # Set validate_state to `False` in Digital Twin mode
+                    validate_state=False if self.shared_utils.digital_twin else Undefined,
```

Comment:
I think we should make a helper function in shared utils like `get_interface_validate_state(self: SharedUtilsProtocol, input: bool | None, peer_in_fabric: bool = False) -> bool | Undefined`

For digital twin it would return false unless the peer is in the fabric. For known peers, if input is false that will win, but not if it is true.
For non-digital-twin it will follow the input if not None, and otherwise Undefined.